test/fixtures/browser-app/app.js

```diff
@@ -7,4 +7,6 @@ var loopback = require('loopback');
 var boot = require('../../../');
 
 var app = module.exports = loopback();
-boot(app);
+app.start = function(done) {
+  boot(app, __dirname, done);
+};
```

Comment:
Ditto, please revert. IMO, when you `require('./test/fixtures/browser-app')`, you should get back an application that is already bootstrapped. More importantly, we need a way how to get a bootstrapped application that is not listening on any http/https port, this is commonly used when generating swagger, an AngularJS client, etc.